lib/StripeMethod.basic.js

```diff
@@ -54,39 +53,60 @@ module.exports = {
     var urlData = this.createUrlData();
     var path = this.createFullPath('/' + id, urlData);
 
-    return utils.callbackifyPromiseWithTimeout(new Promise((function(resolve, reject) {
-      if (isNull) {
-        // Reset metadata:
-        sendMetadata(null, auth);
-      } else if (!isObject) {
-        // Set individual metadata property:
-        var metadata = {};
-        metadata[key] = value;
-        sendMetadata(metadata, auth);
-      } else {
-        // Set entire metadata object after resetting it:
-        this._request('POST', null, path, {
-          metadata: null,
-        }, auth, {}, function(err, response) {
-          if (err) {
-            return reject(err);
+    return utils.callbackifyPromiseWithTimeout(
+      new Promise(
+        function(resolve, reject) {
+          if (isNull) {
+            // Reset metadata:
+            sendMetadata(null, auth);
+          } else if (!isObject) {
+            // Set individual metadata property:
+            var metadata = {};
+            metadata[key] = value;
+            sendMetadata(metadata, auth);
+          } else {
+            // Set entire metadata object after resetting it:
+            this._request(
+              'POST',
+              null,
+              path,
+              {
+                metadata: null,
```

Comment:
minor, would you mind removing the newline before `metadata` here such that the output is `{metadata: null}`? this is the ~one place that prettier respects original formatting, annoyingly

Reply:
(same below)